test/unity/toolkits/drawing_classifier/test_dc_serialization.cxx

```diff
@@ -264,6 +267,8 @@ BOOST_AUTO_TEST_CASE(test_save_load) {
   constexpr unsigned int num_classes = 10;
   const std::string target = "target";
   const std::vector<std::string> features = {"0"};
+  // dc enforces the input feature to be 1, in case someday we want more
```

Comment:
`dc` -> `DC' or even better type it all out: `Drawing Classifier`.

Reply:
I will give you that! Thanks for point out.